### PR TITLE
Update to 1.0

### PR DESCRIPTION
**Pull request by melindaloubser1**
Update to Rasa XE 1.0.1
Database has already been manually migrated to Postgres 12.9

### CODE REVIEW

File: .github/deployments/helmfile.yaml [hunk 1]
```diff
@@ -59,6 +59,10 @@ releases:
                 name: {{ requiredEnv "ACTION_SERVER_SECRET_NAME" }}
                 key: "ALGOLIA_DOCS_INDEX"
       - postgresql:
+          tag: "12.9.0"
+          extraEnv:
+            - name: PGDATA
```
Comment (hsm207) [inline .github/deployments/helmfile.yaml:64]: Why do we need this env var now?

File: .github/deployments/helmfile.yaml [hunk 2]
```diff
@@ -111,8 +116,9 @@ releases:
               session_persistence: true
             rest: {}
       - rabbitmq:
-          rabbitmq:
+          auth:
             password: {{ requiredEnv "RABBITMQ_PASSWORD" }}
+            erlangCookie: {{ requiredEnv "RABBITMQ_ERLANGCOOKIE" }}
```
Comment (hsm207) [inline .github/deployments/helmfile.yaml:121]: Why do we need this env var now?

Comment (melindaloubser1) [reply .github/deployments/helmfile.yaml:121]: The updated Rabbit chart requires it

File: .github/deployments/helmfile.yaml (hunk 1 above)
Comment (melindaloubser1) [reply .github/deployments/helmfile.yaml:64]: For the manual db migration to 12.9 it was required; I tried removing it subsequently but that makes it error out

**Review by hsm207 — APPROVED** [review]

**Review by hsm207 — APPROVED** [review]

**Pull request merged**